docs/application/native/api/iot-headed/4.0/ecore_thread_example_8c-example.html

```diff
@@ -439,21 +439,21 @@
    <span class="keywordflow">if</span> (!path_list)
      {
         Feedback_Thread_Data *ftd;
-        <a name="a43"></a><a class="code" href="group__Ecore__Thread__Group.html#ga4e9de7492f8400e19ddd157d8cc21811">ecore_thread_global_data_add</a>(<span class="stringliteral">&quot;count&quot;</span>, (<span class="keywordtype">void</span> *)3, NULL, <a class="code" href="group__Eina__Types__Group.html#ga1feb115f8e9913e806e090d9bd5a7301">EINA_FALSE</a>);
+        <a name="a37"></a><a class="code" href="group__Ecore__Thread__Group.html#ga4e9de7492f8400e19ddd157d8cc21811">ecore_thread_global_data_add</a>(<span class="stringliteral">&quot;count&quot;</span>, (<span class="keywordtype">void</span> *)3, NULL, <a class="code" href="group__Eina__Types__Group.html#ga1feb115f8e9913e806e090d9bd5a7301">EINA_FALSE</a>);
         ftd = calloc(1, <span class="keyword">sizeof</span>(Feedback_Thread_Data));
         ftd-&gt;name = strdup(<span class="stringliteral">&quot;data0&quot;</span>);
         ftd-&gt;base = strdup(<span class="stringliteral">&quot;/usr/bin&quot;</span>);
-        <a name="a44"></a><a class="code" href="group__Eina__Lock__Group.html#ga8dda398c1db315582a80640c7f78f648" title="Initializes a new #Eina_Lock.">eina_lock_new</a>(&amp;ftd-&gt;mutex);
+        eina_lock_new(&amp;ftd-&gt;mutex);
```

Comment:
I think .. new version is correct, because previous 4.0 headed api ref was made by very old header files.

Reply:
It is same in https://docs1.stg.tizen.org/staging/1358/application/native/api/iot-headed/6.0/ecore_thread_example_8c-example.html, so I asked.

![image](https://user-images.githubusercontent.com/9958374/100174491-e5b82e80-2f0f-11eb-9fb4-2b8da2288a1f.png)

However if you think this issue is not critical, I'll just merge this PR after we get the reply for the dragged window issue.

Reply:
In case of EFL, I will try to fix issues in this page. 
https://code.sec.samsung.net/confluence/pages/viewpage.action?pageId=194846376

Hard to fix it quickly, because EFL team needs to fix upstream header first and sync it to Tizen Platform and SR should be completed. It takes ... at least 2~3 weeks.

Reply:
Yes, correct. I am trying to fix them all.. t.t 
Yesterday, I sent all issues to developers. waiting for fix and feedback..